unit/analyses/constant_propagator.cpp

```diff
@@ -11,17 +11,23 @@ Author: Diffblue Ltd
 
 #include <analyses/constant_propagator.h>
 
-#include <ansi-c/ansi_c_language.h>
-
 #include <goto-programs/goto_convert_functions.h>
 
+#include <util/arith_tools.h>
+#include <util/message.h>
+
 static bool starts_with_x(const exprt &e, const namespacet &)
 {
   if(e.id() != ID_symbol)
     return false;
   return has_prefix(id2string(to_symbol_expr(e).get_identifier()), "x");
 }
 
+static void wrap_with_test(codet &to_wrap, const exprt &test)
```

Comment:
Does this function actually still add value? Doing this one-liner inline might aid clarity.